react/src/index.js

```diff
@@ -45,7 +44,7 @@ if (window.location.hostname === "localhost") {
 
 let BACKEND_URL
 const DSN = process.env.REACT_APP_DSN
-const RELEASE = release("application.monitoring.javascript") || process.env.REACT_APP_RELEASE
```

Comment:
Noting I reviewed this, and I'm pretty sure this `release()` function was stale code (not needed, because process.env.REACT_APP_RELEASE is always defined, assuming you're always running ./run.sh which uses npm run build, or manually running npm run start, instead)

So good call here on simplifying.